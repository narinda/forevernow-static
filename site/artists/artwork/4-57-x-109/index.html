<!doctype html>
<html lang="en" dir="ltr" class="no-js wf-loading">
  <head>
    <meta charset="utf-8">

        <meta http-equiv="Content-type" content="text/html; charset=utf-8" />
    <meta http-equiv="Content-Language" content="en-au" />

    
  <meta name="description" content="As a composer, I am deeply convinced that in every little sound our world has to offer, a symphony is hidden.
It&rsquo;s up to us to lend an ear and deciph"/>
  <meta property="og:url" content="http://forevernow.me/artists/artwork/4-57-x-109/">
  <meta property="og:title" content="4.57 x 10⁹, S U N N S T E D E"/>
  <meta property="og:description" content="As a composer, I am deeply convinced that in every little sound our world has to offer, a symphony is hidden.
It&rsquo;s up to us to lend an ear and decipher these everchanging scores, the same way we pick apples from the tree.
It&rsquo;s a gift from Nature, thus a gift from the Universe.
—
Following that philosophy, « 4.57 x 10⁹ » acts as an earthling soundscape manifesto by mixing raw &amp; primitive elements (birds songs, water &amp; snow noises, …) with contemporary elements (transformed &amp; manipulated guitar takes, apparatus noises, …).
—
« Hello, whoever you are. Those are the kind of sounds and noises that can be heard on our planet… »
"/>
  

  <meta name="twitter:card" content="summary_large_image"/>
  <meta name="twitter:site" content="@aphidslab"/>
  <meta name="twitter:url"  content="http://forevernow.me/artists/artwork/4-57-x-109/"/>
  <meta name="twitter:title" content="4.57 x 10⁹, S U N N S T E D E"/>
  <meta name="twitter:description" content="As a composer, I am deeply convinced that in every little sound our world has to offer, a symphony is hidden.
It&rsquo;s up to us to lend an ear and decipher these everchanging scores, the same way we pick apples from the tree.
It&rsquo;s a gift from Nature, thus a gift from the Universe.
—
Following that philosophy, « 4.57 x 10⁹ » acts as an earthling soundscape manifesto by mixing raw &amp; primitive elements (birds songs, water &amp; snow noises, …) with contemporary elements (transformed &amp; manipulated guitar takes, apparatus noises, …).
—
« Hello, whoever you are. Those are the kind of sounds and noises that can be heard on our planet… »
"/>
  


    <meta name="keywords" content="forever now space voyager aphids mona art interstellar" />
    <meta name="designer" content="Icelab" />
    <meta name="generator" content="Blueprint" />

    
    <link href="../../../static/styles/site-1438647934.css" media="all" rel="stylesheet" type="text/css" />

        <script src="../../../static/scripts/vendor/modernizr.custom.06658-1376354765.js" type="text/javascript"></script>

        <!-- <link rel="icon" href="/static/images/fav.png" type="image/png" /> -->

    <title>Artists - Forever now</title>

    
  
  <script type="text/javascript">
    ENCODE_FORMATS = {"mp4_480p":null,"mp4_baseline_480p":null,"webm_720p":null,"webm_480p":null,"mp4_720p":null,"mp3":"http://assets.forevernow.me/files/a68ebe9b39d5a544a4899c412ad56588.mp3"}
  </script>


  </head>
  <body id="frnvid">
    <div class="body__inner">
      <header class="masthead">
        <div class="masthead__wrapper">
          <a href="../../../index.html" role="banner" class="masthead__banner">
            <h1 class="masthead__title">Forever Now</h1>
          </a>
        </div>
      </header>

      <div class="main-nav">
        <div class="main-nav__wrapper">
          <nav role="navigation">
            <ul>
              
                                        
              <li class="main-nav__list-item"><a href="../../../about.html"><span>About</span></a></li>                          
              <li class="main-nav__list-item--active"><a href="../../../artists.html"><span>Artists</span></a></li>                          
              <li class="main-nav__list-item"><a href="../../../curators.html"><span>Curators </span></a></li>                          
              <li class="main-nav__list-item"><a href="../../../record-selection.html"><span>Record Selection</span></a></li>            
            </ul>
          </nav>
        </div>
      </div>

      










<div class="artwork-content">
  <div class="artwork-content__wrapper">
    <div class="artwork-header">
      <h1 class="artwork-header__title">
        4.57 x 10⁹
      </h1>
      
      
        <h2 class="artwork-header__name">
          S U N N S T E D E 
        </h2>
      
    </div><!-- .artwork-header -->

    <!-- the player -->
    <div class="artwork-player">
      
        <div class="artwork-player__audio">
          <audio>
            <source src="http://assets.forevernow.me/files/a68ebe9b39d5a544a4899c412ad56588.mp3"/>
          </audio>
        </div>
      
    </div><!-- .artwork-player -->

    
      <div class="artwork-description copy">
        <p>As a composer, I am deeply convinced that in every little sound our world has to offer, a symphony is hidden.
It&rsquo;s up to us to lend an ear and decipher these everchanging scores, the same way we pick apples from the tree.
It&rsquo;s a gift from Nature, thus a gift from the Universe.
—
Following that philosophy, « 4.57 x 10⁹ » acts as an earthling soundscape manifesto by mixing raw &amp; primitive elements (birds songs, water &amp; snow noises, …) with contemporary elements (transformed &amp; manipulated guitar takes, apparatus noises, …).
—
« Hello, whoever you are. Those are the kind of sounds and noises that can be heard on our planet… »</p>

      </div>
    

    
      <div class="artwork-bio copy">
        <p>Sound, noise, music — creating with everything because everything is creative.
—
Sunnstede is a one-man experimental musical project.
—
Sunnstede is organic and synthetic, implosive and explosive, raw and transformed, structured and amorphous, electric and acoustic, perpetual and volatile.</p>

      </div>
    

    
<div class="artwork-social">
  
  <a class="twitter-button" href="http://twitter.com/intent/tweet?text=4.57+x+10%E2%81%B9+by+S+U+N+N+S+T+E+D+E&url=http%3A%2F%2Fforevernow.me%2Fartists%2Fartwork%2F4-57-x-109%2F&utm_medium=tweetbutton&utm_source=twitter" target="_blank" data-social='twitter' data-url="http://forevernow.me/artists/artwork/4-57-x-109/">
    <span>
      <i class="icon-twitter"></i>
      Tweet
    </span>
  </a>

  
  <a class="facebook-button" href="http://www.facebook.com/sharer.php?u=http%3A%2F%2Fforevernow.me%2Fartists%2Fartwork%2F4-57-x-109%2F&utm_medium=facebookbutton&utm_source=facebook" target="_blank" data-social='facebook' data-url="http://forevernow.me/artists/artwork/4-57-x-109/">
    <span>
      <i class="icon-facebook-sign"></i>
      Share
    </span>

  
  <a class="pinterest-button" href="http://pinterest.com/pin/create/button/?description=4.57+x+10%E2%81%B9+by+S+U+N+N+S+T+E+D+E&url=http%3A%2F%2Fforevernow.me%2Fartists%2Fartwork%2F4-57-x-109%2F&utm_medium=pinterestbutton&utm_source=pinterest" target="_blank" data-social='pinterest' data-url="http://forevernow.me/artists/artwork/4-57-x-109/">
    <span>
      <i class="icon-pinterest"></i>
      Pin
    </span>
  </a>
</div>


    
      <div class="artwork-shortlist-history">
        
        <div class="artwork-shortlist-history__pole"></div>
        
      </div>
    

    
      


  <div class="artwork-comments" id="comments">
    <h2 class="artwork-comments__header">Comments</h2>
    
      <div class="artwork-comment  launch cmt-index-0" id="cmt650892">
        <div class="artwork-comment__meta">
          <h4 class="artwork-comment__commenter">
            
              <a href="http://www.lothoniel.com" rel="nofollow">Lothoniel</a>
            
          </h4>
          <p class="artwork-comment__date">
            22 May at 10:21PM
          </p>
          
            <div class="artwork-comment__vote">
              <span>Launch</span>
            </div>
          
        </div>
        <div class="artwork-comment__markup">
          <p>Deepening the Journey
<br />To open our conscience.</p>

<p>In every soul & sound experience
<br />Meeting a True Presence.</p>

<p>Of a hidden symphony, the Essence.
<br />When the subtle dances with the wild.</p>

<p>Creating a bridge between the worlds,
<br />On this Path of Remembrance.</p>
        </div>
      </div>
    
      <div class="artwork-comment  launch cmt-index-1" id="cmt662170">
        <div class="artwork-comment__meta">
          <h4 class="artwork-comment__commenter">
            
              <a href="http://evawollenberg.com" rel="nofollow">Wollenberg</a>
            
          </h4>
          <p class="artwork-comment__date">
            31 May at 10:21PM
          </p>
          
            <div class="artwork-comment__vote">
              <span>Launch</span>
            </div>
          
        </div>
        <div class="artwork-comment__markup">
          <p>Exquisite, refined and deep sound ecosystem</p>
        </div>
      </div>
    
      <div class="artwork-comment  launch cmt-index-2" id="cmt919892">
        <div class="artwork-comment__meta">
          <h4 class="artwork-comment__commenter">
            
              <a href="http://silasbe.com" rel="nofollow">Silas B. Ritchie</a>
            
          </h4>
          <p class="artwork-comment__date">
            19 July at 05:21AM
          </p>
          
            <div class="artwork-comment__vote">
              <span>Launch</span>
            </div>
          
        </div>
        <div class="artwork-comment__markup">
          <p>I concur.</p>
        </div>
      </div>
    
  </div>




<div class="artwork-comments-form">
  <h2 class="artwork-comments-form__header">
    
      Join the discussion
      </h2>
  <div class="artwork-comments-form__logged-in-user">
    Comments are now closed.
  </div>
  <div id="comment-errors">
    
  </div>
</div><!-- .artwork-comments-leave -->
    
  </div>
</div>


      <div class="footer">
        <div class="footer__wrapper">
          <div class="footer__submit">
          </div>
          <div class="footer__about">
            <div class="footer__about-description copy">
              <p>Forever Now is an Aphids project. Aphids is an artist-led cultural organisation creating collaborations across artforms and borders. </p>
<p><a href="http://www.aphids.net">www.aphids.net</a></p> 
              <img class="footer__logo" src="../../../static/images/lines.png" alt="">
            </div>
          </div>
          <div class="footer__curator">
            
          </div>
        </div>
      </div>
    </div><!-- .body__inner -->

    <canvas id="starfield" style="background-color:#101010"></canvas>

    <script>
      Modernizr.load([{
        test: Modernizr.fontface,
        yep : '//use.typekit.com/shj1rtn.js',
        callback: function() {
          var removeWfLoading = function() {
            with(document.documentElement) {
              className = className.replace(/\bwf-loading\b/, 'wf-failed');
            }
          };
          setTimeout(function() {
            try { Typekit.load(); } catch(e) { removeWfLoading(); }
          }, 10);
          setTimeout(function() {
            removeWfLoading();
          }, 1000);
        }
      }
    ]);
    </script>
    
  
    <script type="text/javascript">
      Modernizr.load([{
          load: [
            '/static/scripts/vendor/jquery.min.js',
            '/static/scripts/compiled/site.js',
            '/static/scripts/audiojs/audio.min.js',
            '/static/scripts/compiled/audio.js',
            '/static/scripts/vendor/social.js'
          ]
        }, 
        {
          test: Modernizr.canvas, 
          yep : "/static/scripts/compiled/starfieldtest2.js"
        }]);
    </script>
  

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-5902766-4', 'forevernow.me');
      ga('send', 'pageview');

    </script>
  </body>
</html>
