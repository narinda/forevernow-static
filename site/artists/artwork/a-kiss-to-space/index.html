<!doctype html>
<html lang="en" dir="ltr" class="no-js wf-loading">
  <head>
    <meta charset="utf-8">

        <meta http-equiv="Content-type" content="text/html; charset=utf-8" />
    <meta http-equiv="Content-Language" content="en-au" />

    
  <meta name="description" content="Images of science and objects ebb and flow as they build relationships to each other. Some juxtapositions have clear relationships some formal other are po"/>
  <meta property="og:url" content="http://forevernow.me/artists/artwork/a-kiss-to-space/">
  <meta property="og:title" content="A Kiss To Space, Kevin H.  Jones"/>
  <meta property="og:description" content="Images of science and objects ebb and flow as they build relationships to each other. Some juxtapositions have clear relationships some formal other are poetic in their final resting state.

Like a Japanese tea ceremony all objects fall into their proper place.
"/>
  
  <meta property="og:image" content="http://forevernow.me/static/files/assets/56364185/9d55e9f4ee20140529-20135-1i02wcm-0_hero.png"/>
  

  <meta name="twitter:card" content="summary_large_image"/>
  <meta name="twitter:site" content="@aphidslab"/>
  <meta name="twitter:url"  content="http://forevernow.me/artists/artwork/a-kiss-to-space/"/>
  <meta name="twitter:title" content="A Kiss To Space, Kevin H.  Jones"/>
  <meta name="twitter:description" content="Images of science and objects ebb and flow as they build relationships to each other. Some juxtapositions have clear relationships some formal other are poetic in their final resting state.

Like a Japanese tea ceremony all objects fall into their proper place.
"/>
  
  <meta name="twitter:image" content="http://forevernow.me/static/files/assets/56364185/9d55e9f4ee20140529-20135-1i02wcm-0_hero.png"/>
  


    <meta name="keywords" content="forever now space voyager aphids mona art interstellar" />
    <meta name="designer" content="Icelab" />
    <meta name="generator" content="Blueprint" />

    
    <link href="../../../static/styles/site-1438647934.css" media="all" rel="stylesheet" type="text/css" />

        <script src="../../../static/scripts/vendor/modernizr.custom.06658-1376354765.js" type="text/javascript"></script>

        <!-- <link rel="icon" href="/static/images/fav.png" type="image/png" /> -->

    <title>Artists - Forever now</title>

    
  
    <link href="../../../static/styles/flowplayer/minimalist-1371254711.css" media="screen" rel="stylesheet" type="text/css" />
  
  <script type="text/javascript">
    ENCODE_FORMATS = {"mp4_720p":"http://assets.forevernow.me/files/0ff42531dbba4ee9b45c921f61af7840.mp4","mp3":null,"mp4_480p":"http://assets.forevernow.me/files/68144a340b8765ae0c4908b140e9dc25.mp4","mp4_baseline_480p":"http://assets.forevernow.me/files/64c9670175bf96eabe586116c504d906.mp4","webm_720p":"http://assets.forevernow.me/files/d5c5cdd209142d918c0acfd2ff71d754.webm","webm_480p":"http://assets.forevernow.me/files/0a7bf040ba399e87ae742024b7bb22f5.webm"}
  </script>


  </head>
  <body id="frnvid">
    <div class="body__inner">
      <header class="masthead">
        <div class="masthead__wrapper">
          <a href="../../../index.html" role="banner" class="masthead__banner">
            <h1 class="masthead__title">Forever Now</h1>
          </a>
        </div>
      </header>

      <div class="main-nav">
        <div class="main-nav__wrapper">
          <nav role="navigation">
            <ul>
              
                                        
              <li class="main-nav__list-item"><a href="../../../about.html"><span>About</span></a></li>                          
              <li class="main-nav__list-item--active"><a href="../../../artists.html"><span>Artists</span></a></li>                          
              <li class="main-nav__list-item"><a href="../../../curators.html"><span>Curators </span></a></li>                          
              <li class="main-nav__list-item"><a href="../../../record-selection.html"><span>Record Selection</span></a></li>            
            </ul>
          </nav>
        </div>
      </div>

      










<div class="artwork-content">
  <div class="artwork-content__wrapper">
    <div class="artwork-header">
      <h1 class="artwork-header__title">
        A Kiss To Space
      </h1>
      
      
        <h2 class="artwork-header__name">
          Kevin H.  Jones 
        </h2>
      
    </div><!-- .artwork-header -->

    <!-- the player -->
    <div class="artwork-player">
      
        <style>.artwork-player__video {
            background: #000 url("../../../static/files/assets/56364185/9d55e9f4ee20140529-20135-1i02wcm-0_splash.png") 0 0 no-repeat;
            background-size: 100%;
          }
        </style>
        <div 
          class="artwork-player__video" 
          title="a-kiss-to-space">
        </div>

      
    </div><!-- .artwork-player -->

    
      <div class="artwork-description copy">
        <p>Images of science and objects ebb and flow as they build relationships to each other. Some juxtapositions have clear relationships some formal other are poetic in their final resting state.</p>

<p>Like a Japanese tea ceremony all objects fall into their proper place.</p>

      </div>
    

    
      <div class="artwork-bio copy">
        <p>Kevin H. Jones</p>

<p>Kevin H. Jones was born 1969 in Huntsville Alabama and currently resides in New Orleans, Louisiana. Kevin has degrees from Virginia Commonwealth University (BFA, Painting and Printmaking), The University of Texas at Austin (MFA, Painting) and Yale University  (MFA, Design).</p>

<p>Over the past four years within Kevinâ€™s work, one can see transitions in and synthesis of media including painting, video, physical computing, and more recently, 2-dimensional digital prints.  Through this synthesis of media, the conceptual investigation of the natural world through charts, diagrams and systems is a constant theme. His early work used solar energy to power a fictional television station, while more recent work uses sensors to create an interactive video installation that questions entropy.</p>

<p>Kevin has exhibited throughout the United States, Asia and Europe and his work has been featured in ID Magazine, Idea Magazine, Neural Online, The New York Times and MSNBC. Most recently, he has exhibited his work at the Thompson Gallery at Furman University in Greenville, South Carolina, Stasjon K in Sandnes, Norway ,Ginza Geijutsu Laboratory in Tokyo, Japan and 1708 Gallery in Richmond VA.</p>

<p>Kevin was recently awarded Board of Regents ATLAS grant 2011-2012, The Louisiana Division of the Arts Artist Fellowship for 2009-2010; the Louisiana ArtWorks, Studio Residency Program 2009-2010; AIR, Residency, Stasjon K, Sandnes, Norway 2008; the KAT Fund, Houston Contemporary  Art Museum 2006 and two Virginia Museum of Fine Arts Fellowships 1995, 2000.</p>

<p>He has taught at The University of North Carolina at Greensboro, The University of Oregon, The University of Pennsylvania and currently is an Associate Professor in the Newcomb Art Department at Tulane University.</p>

      </div>
    

    
<div class="artwork-social">
  
  <a class="twitter-button" href="http://twitter.com/intent/tweet?text=A+Kiss+To+Space+by+Kevin+H.++Jones&url=http%3A%2F%2Fforevernow.me%2Fartists%2Fartwork%2Fa-kiss-to-space%2F&utm_medium=tweetbutton&utm_source=twitter" target="_blank" data-social='twitter' data-url="http://forevernow.me/artists/artwork/a-kiss-to-space/">
    <span>
      <i class="icon-twitter"></i>
      Tweet
    </span>
  </a>

  
  <a class="facebook-button" href="http://www.facebook.com/sharer.php?u=http%3A%2F%2Fforevernow.me%2Fartists%2Fartwork%2Fa-kiss-to-space%2F&utm_medium=facebookbutton&utm_source=facebook" target="_blank" data-social='facebook' data-url="http://forevernow.me/artists/artwork/a-kiss-to-space/">
    <span>
      <i class="icon-facebook-sign"></i>
      Share
    </span>

  
  <a class="pinterest-button" href="http://pinterest.com/pin/create/button/?description=A+Kiss+To+Space+by+Kevin+H.++Jones&url=http%3A%2F%2Fforevernow.me%2Fartists%2Fartwork%2Fa-kiss-to-space%2F&utm_medium=pinterestbutton&utm_source=pinterest" target="_blank" data-social='pinterest' data-url="http://forevernow.me/artists/artwork/a-kiss-to-space/">
    <span>
      <i class="icon-pinterest"></i>
      Pin
    </span>
  </a>
</div>


    
      <div class="artwork-shortlist-history">
        
        <div class="artwork-shortlist-history__pole"></div>
        
      </div>
    

    
      


  <div class="artwork-comments" id="comments">
    <h2 class="artwork-comments__header">Comments</h2>
    
      <div class="artwork-comment  launch cmt-index-0" id="cmt936585">
        <div class="artwork-comment__meta">
          <h4 class="artwork-comment__commenter">
            
              David Drexler
            
          </h4>
          <p class="artwork-comment__date">
            22 July at 07:30AM
          </p>
          
            <div class="artwork-comment__vote">
              <span>Launch</span>
            </div>
          
        </div>
        <div class="artwork-comment__markup">
          <p>Nicely done!</p>
        </div>
      </div>
    
      <div class="artwork-comment   cmt-index-1" id="cmt1335066">
        <div class="artwork-comment__meta">
          <h4 class="artwork-comment__commenter">
            
              Brian Ritchie
            
          </h4>
          <p class="artwork-comment__date">
            18 September at 02:43PM
          </p>
          
        </div>
        <div class="artwork-comment__markup">
          <p>Simple and effective.</p>
        </div>
      </div>
    
  </div>




<div class="artwork-comments-form">
  <h2 class="artwork-comments-form__header">
    
      Join the discussion
      </h2>
  <div class="artwork-comments-form__logged-in-user">
    Comments are now closed.
  </div>
  <div id="comment-errors">
    
  </div>
</div><!-- .artwork-comments-leave -->
    
  </div>
</div>


      <div class="footer">
        <div class="footer__wrapper">
          <div class="footer__submit">
          </div>
          <div class="footer__about">
            <div class="footer__about-description copy">
              <p>Forever Now is an Aphids project. Aphids is an artist-led cultural organisation creating collaborations across artforms and borders. </p>
<p><a href="http://www.aphids.net">www.aphids.net</a></p> 
              <img class="footer__logo" src="../../../static/images/lines.png" alt="">
            </div>
          </div>
          <div class="footer__curator">
            
          </div>
        </div>
      </div>
    </div><!-- .body__inner -->

    <canvas id="starfield" style="background-color:#101010"></canvas>

    <script>
      Modernizr.load([{
        test: Modernizr.fontface,
        yep : '//use.typekit.com/shj1rtn.js',
        callback: function() {
          var removeWfLoading = function() {
            with(document.documentElement) {
              className = className.replace(/\bwf-loading\b/, 'wf-failed');
            }
          };
          setTimeout(function() {
            try { Typekit.load(); } catch(e) { removeWfLoading(); }
          }, 10);
          setTimeout(function() {
            removeWfLoading();
          }, 1000);
        }
      }
    ]);
    </script>
    
  
    <script type="text/javascript">
      Modernizr.load([{
          load: [
            '/static/scripts/vendor/jquery.min.js',
            '/static/scripts/compiled/site.js',
            '/static/scripts/flowplayer/flowplayer.min.js',
            '/static/scripts/compiled/video.js',
            '/static/scripts/vendor/social.js'
          ]
        }, 
        {
          test: Modernizr.canvas, 
          yep : "/static/scripts/compiled/starfieldtest2.js"
        }]);
    </script>
  

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-5902766-4', 'forevernow.me');
      ga('send', 'pageview');

    </script>
  </body>
</html>
