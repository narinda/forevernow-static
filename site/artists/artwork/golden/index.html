<!doctype html>
<html lang="en" dir="ltr" class="no-js wf-loading">
  <head>
    <meta charset="utf-8">

        <meta http-equiv="Content-type" content="text/html; charset=utf-8" />
    <meta http-equiv="Content-Language" content="en-au" />

    
  <meta name="description" content="A hypnotic sweeping call rising from a secret hidden place, Golden reaches out to its audience and bares a yearning desire to float with them to an ephemer"/>
  <meta property="og:url" content="http://forevernow.me/artists/artwork/golden/">
  <meta property="og:title" content="Golden, Jimmy Tait"/>
  <meta property="og:description" content="A hypnotic sweeping call rising from a secret hidden place, Golden reaches out to its audience and bares a yearning desire to float with them to an ephemeral space, at once vast and lonesome yet intimate and personal.
"/>
  
  <meta property="og:image" content="http://forevernow.me/static/files/assets/6d6d2dda/SARA_FRANKIE-_hero.jpg"/>
  

  <meta name="twitter:card" content="summary_large_image"/>
  <meta name="twitter:site" content="@aphidslab"/>
  <meta name="twitter:url"  content="http://forevernow.me/artists/artwork/golden/"/>
  <meta name="twitter:title" content="Golden, Jimmy Tait"/>
  <meta name="twitter:description" content="A hypnotic sweeping call rising from a secret hidden place, Golden reaches out to its audience and bares a yearning desire to float with them to an ephemeral space, at once vast and lonesome yet intimate and personal.
"/>
  
  <meta name="twitter:image" content="http://forevernow.me/static/files/assets/6d6d2dda/SARA_FRANKIE-_hero.jpg"/>
  


    <meta name="keywords" content="forever now space voyager aphids mona art interstellar" />
    <meta name="designer" content="Icelab" />
    <meta name="generator" content="Blueprint" />

    
    <link href="../../../static/styles/site-1438647934.css" media="all" rel="stylesheet" type="text/css" />

        <script src="../../../static/scripts/vendor/modernizr.custom.06658-1376354765.js" type="text/javascript"></script>

        <!-- <link rel="icon" href="/static/images/fav.png" type="image/png" /> -->

    <title>Artists - Forever now</title>

    
  
    <link href="../../../static/styles/flowplayer/minimalist-1371254711.css" media="screen" rel="stylesheet" type="text/css" />
  
  <script type="text/javascript">
    ENCODE_FORMATS = {"mp4_720p":"http://assets.forevernow.me/files/010fa028cea88d40ca0f355af464292c.mp4","mp3":null,"mp4_480p":"http://assets.forevernow.me/files/b8789a73902c2029ad98a0ddfa756cdc.mp4","mp4_baseline_480p":"http://assets.forevernow.me/files/9e8607e0f40df3e07c9621b90160240e.mp4","webm_720p":"http://assets.forevernow.me/files/818a6be9fd01337d11070ed305f5c286.webm","webm_480p":"http://assets.forevernow.me/files/3c6e1b8e408494232ff1669ea6d00a3d.webm"}
  </script>


  </head>
  <body id="frnvid">
    <div class="body__inner">
      <header class="masthead">
        <div class="masthead__wrapper">
          <a href="../../../index.html" role="banner" class="masthead__banner">
            <h1 class="masthead__title">Forever Now</h1>
          </a>
        </div>
      </header>

      <div class="main-nav">
        <div class="main-nav__wrapper">
          <nav role="navigation">
            <ul>
              
                                        
              <li class="main-nav__list-item"><a href="../../../about.html"><span>About</span></a></li>                          
              <li class="main-nav__list-item--active"><a href="../../../artists.html"><span>Artists</span></a></li>                          
              <li class="main-nav__list-item"><a href="../../../curators.html"><span>Curators </span></a></li>                          
              <li class="main-nav__list-item"><a href="../../../record-selection.html"><span>Record Selection</span></a></li>            
            </ul>
          </nav>
        </div>
      </div>

      










<div class="artwork-content">
  <div class="artwork-content__wrapper">
    <div class="artwork-header">
      <h1 class="artwork-header__title">
        Golden
      </h1>
      
      
        <h2 class="artwork-header__name">
          Jimmy Tait 
        </h2>
      
    </div><!-- .artwork-header -->

    <!-- the player -->
    <div class="artwork-player">
      
        <style>.artwork-player__video {
            background: #000 url("../../../static/files/assets/6d6d2dda/SARA_FRANKIE-_splash.jpg") 0 0 no-repeat;
            background-size: 100%;
          }
        </style>
        <div 
          class="artwork-player__video" 
          title="golden">
        </div>

      
    </div><!-- .artwork-player -->

    
      <div class="artwork-description copy">
        <p>A hypnotic sweeping call rising from a secret hidden place, Golden reaches out to its audience and bares a yearning desire to float with them to an ephemeral space, at once vast and lonesome yet intimate and personal.</p>

      </div>
    

    
      <div class="artwork-bio copy">
        <p>Melbourne 5-piece Jimmy Tait (none of whom are actually called Jimmy) offer dark melancholic pop songs. Since 2008 singer and songwriter Sara Retallick has been working under the name &ndash; a homage to her late grandfather Mr James Tait. The band swerve through genres; angular guitar driven noise floats in to sparsity and space. With a distinctive take on Australian Gothic, Jimmy Tait move from dark melodic ballads to dreamy pop songs. They sound uniquely Australian, with European undertones that only Melbourne could inspire.</p>

      </div>
    

    
<div class="artwork-social">
  
  <a class="twitter-button" href="http://twitter.com/intent/tweet?text=Golden+by+Jimmy+Tait&url=http%3A%2F%2Fforevernow.me%2Fartists%2Fartwork%2Fgolden%2F&utm_medium=tweetbutton&utm_source=twitter" target="_blank" data-social='twitter' data-url="http://forevernow.me/artists/artwork/golden/">
    <span>
      <i class="icon-twitter"></i>
      Tweet
    </span>
  </a>

  
  <a class="facebook-button" href="http://www.facebook.com/sharer.php?u=http%3A%2F%2Fforevernow.me%2Fartists%2Fartwork%2Fgolden%2F&utm_medium=facebookbutton&utm_source=facebook" target="_blank" data-social='facebook' data-url="http://forevernow.me/artists/artwork/golden/">
    <span>
      <i class="icon-facebook-sign"></i>
      Share
    </span>

  
  <a class="pinterest-button" href="http://pinterest.com/pin/create/button/?description=Golden+by+Jimmy+Tait&url=http%3A%2F%2Fforevernow.me%2Fartists%2Fartwork%2Fgolden%2F&utm_medium=pinterestbutton&utm_source=pinterest" target="_blank" data-social='pinterest' data-url="http://forevernow.me/artists/artwork/golden/">
    <span>
      <i class="icon-pinterest"></i>
      Pin
    </span>
  </a>
</div>


    
      <div class="artwork-shortlist-history">
        
        <div class="artwork-shortlist-history__pole"></div>
        
      </div>
    

    
      





<div class="artwork-comments-form">
  <h2 class="artwork-comments-form__header">
    
      Start the discussion
      </h2>
  <div class="artwork-comments-form__logged-in-user">
    Comments are now closed.
  </div>
  <div id="comment-errors">
    
  </div>
</div><!-- .artwork-comments-leave -->
    
  </div>
</div>


      <div class="footer">
        <div class="footer__wrapper">
          <div class="footer__submit">
          </div>
          <div class="footer__about">
            <div class="footer__about-description copy">
              <p>Forever Now is an Aphids project. Aphids is an artist-led cultural organisation creating collaborations across artforms and borders. </p>
<p><a href="http://www.aphids.net">www.aphids.net</a></p> 
              <img class="footer__logo" src="../../../static/images/lines.png" alt="">
            </div>
          </div>
          <div class="footer__curator">
            
          </div>
        </div>
      </div>
    </div><!-- .body__inner -->

    <canvas id="starfield" style="background-color:#101010"></canvas>

    <script>
      Modernizr.load([{
        test: Modernizr.fontface,
        yep : '//use.typekit.com/shj1rtn.js',
        callback: function() {
          var removeWfLoading = function() {
            with(document.documentElement) {
              className = className.replace(/\bwf-loading\b/, 'wf-failed');
            }
          };
          setTimeout(function() {
            try { Typekit.load(); } catch(e) { removeWfLoading(); }
          }, 10);
          setTimeout(function() {
            removeWfLoading();
          }, 1000);
        }
      }
    ]);
    </script>
    
  
    <script type="text/javascript">
      Modernizr.load([{
          load: [
            '/static/scripts/vendor/jquery.min.js',
            '/static/scripts/compiled/site.js',
            '/static/scripts/flowplayer/flowplayer.min.js',
            '/static/scripts/compiled/video.js',
            '/static/scripts/vendor/social.js'
          ]
        }, 
        {
          test: Modernizr.canvas, 
          yep : "/static/scripts/compiled/starfieldtest2.js"
        }]);
    </script>
  

    <script>
      (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
      (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
      m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
      })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

      ga('create', 'UA-5902766-4', 'forevernow.me');
      ga('send', 'pageview');

    </script>
  </body>
</html>
